scaleway/resource_compute_instance_placement_group.go

```diff
@@ -0,0 +1,146 @@
+package scaleway
+
+import (
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/scaleway/scaleway-sdk-go/api/instance/v1"
+)
+
+func resourceScalewayComputeInstancePlacementGroup() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceScalewayComputeInstancePlacementGroupCreate,
+		Read:   resourceScalewayComputeInstancePlacementGroupRead,
+		Update: resourceScalewayComputeInstancePlacementGroupUpdate,
+		Delete: resourceScalewayComputeInstancePlacementGroupDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+		SchemaVersion: 0,
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+				Description: "The name of the placement group",
+			},
+			"policy_type": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Default:     instance.ComputeClusterPolicyTypeLowLatency.String(),
```

Comment:
I would not put any default value, it's an advance feature and none of the default value make sense.

Reply:
same for `policy_mode`

Reply:
This is already the default on the API side and it's well documented https://developers.scaleway.com/en/products/instance/api/#post-2bb6da